### PR TITLE
#165578239 implement 404 page

### PR DESCRIPTION
**Pull request by thislekan**
#### What does this PR do?

- add 404 error page

#### How should this be manually tested?

If you don't have ```node.js``` installed, please follow this link to [download node.js](https://nodejs.org/en/)

Download the repo by clicking the green ```clone or download``` button above. If you chose the download option, unzip your files into a folder. If you chose the clone option, then run ```git clone 'the-URL-copied-by-clicking-the-green-button'```

Please note: This information assumes a basic understanding of node & git

> Run the following commands in your terminal:

```
# Change directory to clone or zipped folder
cd apollo-ah-frontend (or the name of the folder you zipped the files in)

# Switch to staging branch
git checkout staging
git checkout feature/165578239/404-page

# Install package modules
npm install

# Start up the server
npm run start:dev

- navigate to an unknown route like this: `/hshjkj/`

```

#### What are the relevant pivotal tracker stories?

[#165578239](https://www.pivotaltracker.com/story/show/165578239)

### CODE REVIEW

**Review by valenstical — APPROVED** [review]
LGTM

File: src/test/views/PageLoader.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import PageLoader from '../../views/PageLoader';
+
+describe('<PageLoader />', () => {
+  it('should render the PagLoader', () => {
```
Comment (aanchirinah) [inline src/test/views/PageLoader.test.js]: #### Nitpick
```suggestion
  it('should render the PageLoader', () => {
```

**Review by aanchirinah — CHANGES_REQUESTED** [review]
Awesome work , just correct the typo I pointed out.

File: src/test/views/PageLoader.test.js (hunk 1 above)
Comment (valenstical) [reply src/test/views/PageLoader.test.js]: Thanks @aanchirinah , this has been fixed now.

**Review by aanchirinah — APPROVED** [review]

**Pull request merged**